### PR TITLE
GLTFLoader: No mesh clone if it is referenced only once

### PR DESCRIPTION
**Pull request by takahirox**
I don't think we need to clone mesh if it's referenced only once.

### CODE REVIEW

File: examples/js/loaders/GLTFLoader.js [hunk 1]
```diff
@@ -2940,36 +2940,23 @@ THREE.GLTFLoader = ( function () {
...
+					// for Specular-Glossiness
+					node.onBeforeRender = mesh.onBeforeRender;
 
-						if ( child.material && child.material.isGLTFSpecularGlossinessMaterial === true ) {
+					for ( var i = 0, il = node.children.length; i < il; i ++ ) {
 
-							node.children[ i ].onBeforeRender = child.onBeforeRender;
-
-						}
+						node.children[ i ].onBeforeRender = mesh.children[ i ].onBeforeRender;
```
Comment (donmccurdy) [inline examples/js/loaders/GLTFLoader.js]: Probably we should append `_instance_<N>` to the child names here, too.

Comment (takahirox) [reply examples/js/loaders/GLTFLoader.js]: Good catch, thanks.

**Comment by mrdoob** [comment]
@donmccurdy looks good?

**Review by donmccurdy — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**